include/flyft/grid_interpolator.h

```diff
@@ -0,0 +1,31 @@
+#ifndef FLYFT_GRID_INTERPOLATOR_H_
+#define FLYFT_GRID_INTERPOLATOR_H_
+
+#include "flyft/three_dimension_index.h"
+#include <iostream>
+#include <tuple>
+#include <cmath>
```

Comment:
This is the only header you need here from the standard library, right? Include the other stuff in the `.cc` file where it's used.
```suggestion
#include <string>
```